stdlib/public/Python/Python.swift

```diff
@@ -215,9 +215,20 @@ public enum PythonError : Error, Equatable {
 extension PythonError : CustomStringConvertible {
   public var description: String {
     switch self {
-    case .exception(let p): return "Python exception: \(p)"
-    case .invalidCall(let p): return "Invalid Python call: \(p)"
-    case .invalidModule(let m): return "Invalid Python module: \(m)"
+    case .exception(let e, let t):
+      var exceptionDescription = "Python exception: \(e)"
+      if let t = t {
+        let traceback = Python.import("traceback")
+        exceptionDescription += """
+        \nTraceback:
```

Comment:
It would make it slightly more readable if these 3 lines get indented, because lines 222-225 form a single expression.